### PR TITLE
Button patch

### LINKED ISSUE #1073: Share button is highlighted by default
**Actual Behaviour**

The share button is highlighted by default the moment 'Share on social media' tab is opened

**Expected Behaviour**

The button should not be implicitly highlighted unless hovered/tab pressed upon.

**Steps to reproduce it**

Open the 'Share on social media' tab and observe the 'share on google+' button. It is highlighted unlike the other two share buttons.

**Screenshots of the issue**

screen shot 2018-01-28 at 6 24 09 pm

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by VibhorCodecianGupta**
Fixes #1073 

Changes: Share buttons are not highlighted by default, unless hovered/tabbed upon
screen shot 2018-01-28 at 8 17 15 pm

Demo Link: knotty-chairs.surge.sh

Screenshots for the change:

### CODE REVIEW

File: src/components/ChatApp/Settings/Settings.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .tabStyle{
-  height: 550px;
+  height: auto;
```
Comment (isuruAb) [inline src/components/ChatApp/Settings/Settings.css]: could you please explain this ?

Comment (VibhorCodecianGupta) [reply src/components/ChatApp/Settings/Settings.css]: This change was made in a commit for #1017 . I branched into button-patch at this point, so this change came in here as well. For that particular issue, this change makes the left tab in settings page adjust the height according to the amount of content to be displayed.
8 tabs when looged out, hence 400px; 11 tabs when logged in, hence 550px.

**Comment by VibhorCodecianGupta** [comment]
Squashed commits, please review :)

**Comment by VibhorCodecianGupta** [comment]
No sir, that addition is a result of squashing the commits together, hence the pulled changes are also present in this commit

**Comment by VibhorCodecianGupta** [comment]
Used a `git revert` to revert the squashed commit, then made the concerned changes only 
Please review :)

**Comment by VibhorCodecianGupta** [comment]
Travis fixed, please review :)

**Comment by VibhorCodecianGupta** [comment]
Please check #1097 
Due to the git history getting messy (because of the presence of reverted commits) squashing and rebasing was becoming a bad idea for the commit history. Hence a fresh PR addressing the same issue.

Shall i close this PR?

**Pull request closed without merging**